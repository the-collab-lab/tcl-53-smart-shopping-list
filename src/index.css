/* Custom Fonts */
@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@100;400;700&display=swap');
@font-face {
	font-family: 'Psychofun';
	font-style: normal;
	font-weight: 400;
	src: url('../public/fonts/PsychofunRegular-vmBXy.ttf');
}

@tailwind base;
@tailwind components;
@tailwind utilities;
