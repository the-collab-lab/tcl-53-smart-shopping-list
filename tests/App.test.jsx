// import { render } from '@testing-library/react';
// import { App } from '../src/App';

// it('renders without crashing', () => {
// 	render(<App />);
// });

import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { App } from '../src/App';

afterEach(cleanup);

it('should take a snapshot', () => {
	const { asFragment } = render(<App />);

	expect(asFragment(<App />)).toMatchSnapshot();
});
