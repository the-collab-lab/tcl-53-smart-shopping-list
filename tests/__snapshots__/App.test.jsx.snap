// Vitest Snapshot v1

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="Layout"
  >
    <header
      class="Layout-header"
    >
      <h1>
        Smart shopping list
      </h1>
    </header>
    <main
      class="Layout-main"
    >
      <div
        class="Home"
      >
        <p>
          Hello from the home (
          <code>
            /
          </code>
          ) page!
        </p>
        <button>
          Create a new list
        </button>
      </div>
    </main>
    <nav
      class="Nav"
    >
      <a
        aria-current="page"
        class="Nav-link active"
        href="/"
      >
        Home
      </a>
      <a
        class="Nav-link"
        href="/list"
      >
        List
      </a>
      <a
        class="Nav-link"
        href="/add-item"
      >
        Add Item
      </a>
    </nav>
  </div>
</DocumentFragment>
`;
